### DISCUSSION TITLE
Bag of Words. Unknown words

### DISCUSSION THREAD

**Discussion by Sequner** [seominjoon/kaist-ai605#122] [Q&A] [answered]
Hello Professor and TAs,

What should we do with the words that are not in the vocabulary? As professor mentioned, "unk" token will only degrade the performance. So, should we just ignore those words?

**Comment by seominjoon** [comment, marked as answer]
You should just ignore those words if they are not in your vocab, instead of mapping to UNK. This is a duplicate with #116 .
